docs/guides/genomics-platform/requesting-data/about-our-data.md

```diff
@@ -50,9 +50,9 @@ Somatic VCF files available on St. Jude Cloud have been generated when possible
 
 The validation_status field indicates the status of validating the presence and origin (somatic/germline) of the variant using an orthogonal methodology. Note that this doesn't imply anything regarding variant significance, pathogenicity, or relation to disease. Typically, validation is performed either by secondary sequencing using a custom capture panel designed around the called variants (common in research studies) or by cross-validation using simultaneous sequencing by WGS, WES, and/or RNA-Seq (used in clinical genomics).
  
-For validation by a secondary method, "valid" indicates the variant was validated by the secondary method, "putative" indicates that the secondary method could neither confirm nor invalidate the variant (e.g. due to lack of read coverage by the secondary method), and "invalid" indicates a negative result from the secondary method.
+Validation by a secondary sequencing method has commonly been used in retrospective research studies such as PCGP. Here, "valid" indicates the variant was validated by the secondary method, "putative" indicates that the secondary method could neither confirm nor invalidate the variant (e.g. due to lack of read coverage by the secondary method), and "invalid" indicates a negative result from the secondary method.
```

Comment:
I agree with @adthrasher here (although I initially didn't like "method" either). But I like the other additions. How about just removing the word "sequencing"?
```suggestion
Validation by a secondary method has commonly been used in retrospective research studies such as PCGP. Here, "valid" indicates the variant was validated by the secondary method, "putative" indicates that the secondary method could neither confirm nor invalidate the variant (e.g. due to lack of read coverage by the secondary method), and "invalid" indicates a negative result from the secondary method.
```

Reply:
I don't think we want to reference PCGP either. All of our studies use a secondary method for validation. I think the way this is worded, the paragraph reads like it is mostly related to PCGP, when it should be a general description of validation methodology.  Additionally, our clinical studies use a secondary method for validation, so this isn't exclusive to research studies.